src/Admin/BaseFieldDescription.php

```diff
@@ -297,14 +297,14 @@ public function getFieldValue($object, $fieldName)
             if (method_exists($object, $getter) && \is_callable([$object, $getter])) {
                 $this->cacheFieldGetter($object, $fieldName, 'getter', $getter);
 
-                return \call_user_func_array([$object, $getter], $parameters);
+                return $object->{$getter}(...$parameters);
             }
         }
 
         if (method_exists($object, '__call')) {
             $this->cacheFieldGetter($object, $fieldName, 'call');
 
-            return \call_user_func_array([$object, '__call'], [$fieldName, $parameters]);
+            return $object->__call($fieldName, $parameters);
```

Comment:
Isn't that the same as `$object->$fieldname($parameters);` ?

Reply:
I think that the answer is yes, but then why the previous call was not `\call_user_func_array([$object, $fieldName], $parameters);`?
Note that the check is not `method_exists($object, $fieldName)`.

Reply:
I think `method_exists($object, $fieldName)` will return `true` even if the method is not publicly accessible.

Reply:
yeah, but `$fieldName` is not a method name, is it? It's the name of a field, right?

Reply:
I await for your comments: https://3v4l.org/16GNR.

Reply:
Isn't this a more realistic scenario of would happen here: https://3v4l.org/Bqn9Z ?

Reply:
I guess, so I think we should preserve the call to `__call()` in order to delegate the responsibility to its implementation.

Reply:
I really don't see the point of keeping `__call` when you could write `$fieldName`, but as you wish…